mmv1/third_party/terraform/tests/resource_sql_database_instance_test.go.erb

```diff
@@ -31,6 +31,14 @@ var ignoredReplicaConfigurationFields = []string{
 	"deletion_protection",
 }
 
+var ignoredSourceRepresentationFields = []string{
+	"ca_certificate",
```

Comment:
Assuming these should be snake case based on the above lines.

Reply:
Yep!